tests-performance/src/template-with-list-of-automata.cc

```diff
@@ -0,0 +1,139 @@
+#include <mata/inter-aut.hh>
+#include <mata/nfa.hh>
+#include <mata/mintermization.hh>
+#include <iostream>
+#include <fstream>
+#include <chrono>
+#include <string>
+#include <cstring>
+
+using namespace Mata::Nfa;
+
+int main(int argc, char *argv[])
+{
+    /**
+     * Comment out automata, that you do not want to process

```

Comment:
Suggest using this template for your own benchmark automata.
```suggestion
     * Comment out automata, that you do not want to process or add your own automata.

```